api/HTMLSpanElement.json

```diff
@@ -23,10 +23,10 @@
             "version_added": true
           },
           "opera": {
-            "version_added": true
+            "version_added": "15"
```

Comment:
Interesting. `<span>` is ancient, but in Presto it was a plain `HTMLElement`, not `HTMLSpanElement`. So this is correct.